test/specs/components/List/List-test.tsx

```diff
@@ -72,5 +72,22 @@ describe('List', () => {
       expect(updatedItems.at(0).props().selected).toBe(false)
       expect(updatedItems.at(1).props().selected).toBe(true)
     })
+
+    it('calls onClick handler for item if `selectable`', () => {
+      const onClick = jest.fn()
+      const onSelectedIndexChange = jest.fn()
+      const listItems = mountWithProvider(
+        <List items={getItems(onClick)} onSelectedIndexChange={onSelectedIndexChange} selectable />,
+      ).find('ListItem')
+
+      listItems
+        .first()
+        .find('li')
+        .first()
+        .simulate('click')
+
+      expect(onClick).toHaveBeenCalled()
```

Comment:
Don't you want to check cb params as well?

Reply:
Done 👍